truckersmp_cli/args.py

```diff
@@ -59,6 +57,9 @@ def check_args_errors():
 making sure it's the same directory as Proton""")
             Args.prefixdir = os.path.join(Args.prefixdir, "pfx")
 
+        # Always activate the Windows Steam check when not using Proton
+            Args.check_windows_steam = True
```

Comment:
* Start with lowercase (Coding style for comments)
* Fix the indentation
```suggestion
        # always activate the Windows Steam check when not using Proton
        Args.check_windows_steam = True
```